PR: State of the Spyder Part 2: Looking up

CAM-Gerlach opened the pull request:
New blog post, the second in the series. Somewhat shorter and tighter than the first, focusing on just the major items/accomplishments, with followups to handle the details. Only included two images (only a few paragraphs apart :/ ) previewing the new docs and some Spyder 4 features. I could put in website screenshots from NumFOCUS and Quansight, but didn't think they'd add much over just visiting the linked pages. Also, could put in a screenshot of the one most visible new feature in Spyder 3.3, the all new error dialog, but thought that might create a bad impression since it suggests errors with Spyder. Which we all know, it doesn't have any :)

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+body:
+
+After sharing some major milestones, development progress, and other tidbits from the past six months in [Part 1](https://www.spyder-ide.org/blog/spyder-status-2018-past/) of this series (check that one out first if you haven't already), we now have some amazing news to share with you all here in Part 2, along with other status updates.
+That's not all, thought—Part 3 will look ahead toward Spyder 4 and beyond, unveiling and explaining our full roadmap and going over the future possibilities even further afield.
```
though

CAM-Gerlach replied:
Yup, silly typo. Fixed.

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+
+## Spyder Wins NumFOCUS Development Grant
+
+First up, we're thrilled to announce a major part of what's making that possible (along with your support, of course!).
```
I don't think it's clear what "that" refers to

CAM-Gerlach replied:
Good point. Changed to "that plan".

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+## Spyder Wins NumFOCUS Development Grant
+
+First up, we're thrilled to announce a major part of what's making that possible (along with your support, of course!).
+Spyder was just awarded a $3000 development grant from [NumFOCUS](https://www.numfocus.org/) an organization promoting better science through open code, to help with finishing Spyder 4!
```
comma after NumFOCUS? Or perhaps you can remove the phrase "an organization ..." because you explain it in the next sentences? also, depending on when this actually gets published, "just" may be a bit of a stretch ...

CAM-Gerlach replied:
Yup; a comma is the much better choice. Well, I didn't think it would take two and a half (and counting) weeks to get reviewed and approved... But I guess it could still be justified, since even that time is small compared to the over half a year it lasts.

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+
+First up, we're thrilled to announce a major part of what's making that possible (along with your support, of course!).
+Spyder was just awarded a $3000 development grant from [NumFOCUS](https://www.numfocus.org/) an organization promoting better science through open code, to help with finishing Spyder 4!
+NumFOCUS is a nonprofit dedicated to supporting [key scientific computing projects](https://www.numfocus.org/sponsored-projects); [promoting sustainability](https://www.numfocus.org/programs/sustainability) in the open source ecosystem; educating scientists, engineers, developers and data analysts through their flagship [PyData](https://www.numfocus.org/programs/sustainability) program; and enabl ...
```
Not sure I like having several links after each other to the same URL

CAM-Gerlach replied:
I accidentally pasted the same link several times, and one of the other links is incorrect as well. Fixed now to the correct (unique) links, thanks.

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+NumFOCUS is a nonprofit dedicated to supporting [key scientific computing projects](https://www.numfocus.org/sponsored-projects); [promoting sustainability](https://www.numfocus.org/programs/sustainability) in the open source ecosystem; educating scientists, engineers, developers and data analysts through their flagship [PyData](https://www.numfocus.org/programs/sustainability) program; and enabl ...
+
+We applied for the funding back in March, along with many other scientific python projects, and were one of the fortunate ones selected after the review process.
+The award will run from now until mid-November, and will help allow for multiple betas with new features to be released throughout 2018, with first feature-complete release candidate planned by the year's end (we'll have more detail on all that in Part 3).
```
"will help allow" does not sound right to me. Add "the" before "first"

CAM-Gerlach replied:
Yeah, the first I replaced with "will enable" to make it sound both stronger and less awkward. Yup, clear typo on the second one.

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+
+## Quansight Hires Spyder Maintainer; Helps Fund Development
+
+Equally exciting, Spyder's maintainer, [Carlos Cordoba](https://github.com/ccordoba12), was just hired [Quansight](https://www.quansight.com/), the brand-new company headed by scientific Python luminary [Travis Oliphant](https://twitter.com/teoliphant?lang=en)!
```
hired **by**.

CAM-Gerlach replied:
lol...fixed.

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+
+Equally exciting, Spyder's maintainer, [Carlos Cordoba](https://github.com/ccordoba12), was just hired [Quansight](https://www.quansight.com/), the brand-new company headed by scientific Python luminary [Travis Oliphant](https://twitter.com/teoliphant?lang=en)!
+The father of much of the modern PyData ecosystem including NumPy, SciPy, and Numba, Travis has been instrumental in the dramatic growth of open-source data analysis with Python through founding or helping lead Enthought, Continuum/Anaconda, NumFOCUS, and more.
+Quansight, Travis' newest venture, focuses on scaling up the PyData ecosystem to solve even larger and more integrated problems, connecting companies with with the open-source tools that can benefit their business, and helping build a more robust ecosystem for open-source scientific development.
```
duplicated "with"

CAM-Gerlach replied:
Huh, though I caught that one before, but must have been somewhere else. Fixed.

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+The father of much of the modern PyData ecosystem including NumPy, SciPy, and Numba, Travis has been instrumental in the dramatic growth of open-source data analysis with Python through founding or helping lead Enthought, Continuum/Anaconda, NumFOCUS, and more.
+Quansight, Travis' newest venture, focuses on scaling up the PyData ecosystem to solve even larger and more integrated problems, connecting companies with with the open-source tools that can benefit their business, and helping build a more robust ecosystem for open-source scientific development.
+
+As part of that mission, Carlos was hired through at least the end of 2018 to help several open-source projects sustainable in the long run, and I'm sure you all will be happy to hear that one of them will be our very own Spyder!
```
"to help several open-source projects sustainable" ?

CAM-Gerlach replied:
"help make" ... "more sustainable"

CAM-Gerlach commented:
Thanks @jitseniesen ! A lot of silly typos with this one...I'm like the legendary wizard Merlin; good at finding others' mistakes, even small ones, but blind to my own (unless I do a clean review pass with enough separation from writing the original text).

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+## Spyder 3.3 Almost Ready to Go
+
+Meanwhile, on the development side of things, we're hard at work putting the finishing touches on Spyder 3.3, the next Spyder 3 release.
+We'll have a full post detailing the changes and improvements when it releases, currently planned for June 15.
```
The date needs to change, and other updates may also be needed.

CAM-Gerlach replied:
Thanks; made quite a few improvements in this vein.

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+Meanwhile, on the development side of things, we're hard at work putting the finishing touches on Spyder 3.3, the next Spyder 3 release.
+We'll have a full post detailing the changes and improvements when it releases, currently planned for June 15.
+In brief, it'll have some significant under the hood changes, cleaning house of some old end of life dependencies and legacy support (Python 3.3 and PyQt4/<5.5), thus the bump in minor version number.
+Additionally, it'll add new preferences options for console plotting and autocompletion, improve UX in Variable Explorer editors and greatly improve their performance, add a new, fully automated error reporting feature to make submitting reports faster, easier and more helpful, and contain a number of other minor improvements and usability enhancements, along with numerous bug fixes.
```
perhaps add "user experience" between brackets after "UX" to explain this abbreviation

CAM-Gerlach replied:
Just changed UX -> "user experience" directly

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+We'll be hosting a flash talk at [SciPy 2018](https://scipy2018.scipy.org/ehome/index.php?eventid=299527&), so be sure to catch up with us there.
+
+We're also planning on much more frequent announcements, status reports, news, and tutorials right here on our blog, as well as more timely updates on our social media, including our [Facebook page](https://www.facebook.com/SpyderIDE/notifications/) and [Twitter account](https://twitter.com/Spyder_IDE).
+Make to subscribe/follow if you'd like to stay up to date or want to get in touch; we only post important happenings, so we won't inundate your feeds with trivialities.
```
Make **sure** to ...

CAM-Gerlach replied:
Rewrote (to "We're planning on much more frequent announcements") after deleting the previous out of date line.

jitseniesen commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,92 @@
+Make to subscribe/follow if you'd like to stay up to date or want to get in touch; we only post important happenings, so we won't inundate your feeds with trivialities.
+You can also reach us on our [Google Groups discussion board](https://groups.google.com/forum/#!forum/spyderlib), our [Gitter live chat](https://gitter.im/spyder-ide/public), or post a comment right here below this post.
+
+Thanks again for all you do to make Spyder possible, and keep a look out for the next post in this series on the future of Spyder, along with our three major releases in the next couple weeks. Happy Spydering!
```
couple **of** weeks

CAM-Gerlach replied:
Actually, "couple weeks" is idiomatic (American?) English. However, changed it to the "next few weeks" since I'm not sure of an exact date for 4.0b1 at this point.

CAM-Gerlach commented:
@ccordoba12 Updated with current dates/timeframes and addressed other minor reviewer feedback, so ready when you are. If you can merge soon, I won't need to update it again. Thanks!

goanpeca commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+---
+author: camgerlach
+---
+summary: In the second of a three part series, we've got some exciting announcements to make about Spyder and its continued development, and we'll go over our current status and several releases, projects and events in the near-term pipeline.
```
`go over our current status and several releases` that phrase sounds funny :-p

CAM-Gerlach replied:
Its correct and fairly idiomatic, but it does sound a bit funny. I re-wrote it to

> and we'll review the project's current status and some major upcoming releases.

If that helps.

goanpeca commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+## Spyder Wins NumFOCUS Development Grant
+
+First up, we're thrilled to announce a major part of what's making that plan possible (along with your support, of course!).
+Spyder was just awarded a $3000 development grant from [NumFOCUS](https://www.numfocus.org/), an organization promoting better science through open code, to help with finishing Spyder 4!
```
`... just awarde...` requires a date, since these blogs will live in the time to come

CAM-Gerlach replied:
Changed to "This May, Spyder was awarded...", though more for correctness than anything. The date is on the blog post, so relative dates all have an absolute benchmark, and it is perfectly appropriate for a blog post (particularly an announcement-focused one such as this) to have relative dates since it is primary intended to be contextual snapshot in time, versus say a wiki/encyclopedia article which is a continually updated source of information and without an explicit nor monolithic publication date for the content therein.

goanpeca commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+Spyder was just awarded a $3000 development grant from [NumFOCUS](https://www.numfocus.org/), an organization promoting better science through open code, to help with finishing Spyder 4!
+NumFOCUS is a nonprofit dedicated to supporting [key scientific computing projects](https://www.numfocus.org/sponsored-projects); [promoting sustainability](https://www.numfocus.org/programs/sustainability) in the open source ecosystem; educating scientists, engineers, developers and data analysts through their flagship [PyData](https://www.numfocus.org/programs/pydata) program; and enabling and  ...
+
+We applied for the funding back in March, along with many other scientific python projects, and were one of the fortunate projects selected after the review process.
```
We need to add full dates when describing time moments (March, 2018)

CAM-Gerlach replied:
I added "of this year" for clarity. For the reasons summarized above, including full absolute date components that are the same as the current one (e.g. the current year, in this case) in this sort of context (a dated announcement blog post) is unnecessary, and furthermore is actually sufficiently un-idomatic in English blog writing to the point it could potentially even be actively confusing since readers would likely not expect to see the year written out if it is the current one.

At least, I *hope* it won't take a year to get these blogs posted haha :P

goanpeca replied:
I personally feel annoyed of atemporal expressions like `recently`, `just now` on blogposts, even if the blogpost contains a year. But maybe that is just me 🤷‍♂️

CAM-Gerlach replied:
Well, it depends on the type of blog post, the nature of the event the date/timeframe represents, and the degree of relatively expressed—on that last point, statements like "recently" or particularly "just now" are far more relative and ambiguous than a relative year explicitly stated at the top of the most and likely going to be the current one for the vast majority of likely readers. On those that are likely to be useful to a substantial percentage of users for future reference, like howtos, tutorials, and other explanatory/informative content, then it would be problematic, and you could make a good argument even for news blogs as well, and decisively so on any reference material, like Wikipedia or even our own wiki. 

However, for status updates like this, where a primary goal is to *emphasize* the currency of the information, and that lose the vast majority of their relevance and readership by the end of the relative time period in question, any minor convenience gain from repeating the absolute year is greatly outweighed, I would argue, by how strange, unexpected and possibly even confusing it would sound to many readers used to the conventions in these types of posts, as well as making those dates sound less current and grounded in the present. Even for someone reading the post in the future, stating an absolute rather than relative year may be interpreted less naturally than just referencing the current relative year of the post, which a historical reader (being so inclined to look "back in time" in the first place) would already have in mind. 

Furthermore, as a status update, many dates are either natively defined or have their relevance in a relative, rather than absolute frame, and there's also events for which we have no need, or even an active interest in not being overly specific about exact dates. For example, Carlos' officially started his job (IIRC) sometime in May 2018, but it makes more sense in this context to say "recently" since the exact date matters little to our audience, but what is important is that it happened in the recent past and thus we're announcing it now, which is what the word emphasizes. Overall, this is where context, purpose and audience become very important in a language and things can get very tricky from a prescriptive perspective.

CAM-Gerlach replied:
I appreciate having this discussion though @goanpeca since it really made me think through what's behind a lot of decisions that are normally just intuitive and "natural" for me :)

goanpeca commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+NumFOCUS is a nonprofit dedicated to supporting [key scientific computing projects](https://www.numfocus.org/sponsored-projects); [promoting sustainability](https://www.numfocus.org/programs/sustainability) in the open source ecosystem; educating scientists, engineers, developers and data analysts through their flagship [PyData](https://www.numfocus.org/programs/pydata) program; and enabling and  ...
+
+We applied for the funding back in March, along with many other scientific python projects, and were one of the fortunate projects selected after the review process.
+The award will run from now until mid-November, and will enable multiple betas with new features to be released throughout 2018, with the first feature-complete release candidate planned by the year's end (we'll have more detail on all that in Part 3).
```
Same (I know there is a 2018 later on, but... is better to be super explicit I think)

CAM-Gerlach replied:
Changed to "from now through this November". Again, including this year when we're in the middle of it would sound very odd in this context in English, and the blog post is datestamped for those viewing for the purposes of historical research (and that a small minority of likely viewers, for a post like this vs. e.g. a tutorial/howto type post).

goanpeca commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+
+## Quansight Hires Spyder Maintainer; Helps Fund Development
+
+Equally exciting, Spyder's maintainer, [Carlos Cordoba](https://github.com/ccordoba12), was just hired by [Quansight](https://www.quansight.com/), the brand-new company headed by scientific Python luminary [Travis Oliphant](https://twitter.com/teoliphant?lang=en)!
```
`just` Same :-p  (or maybe we can remove it?)

CAM-Gerlach replied:
Good catch; changed to "recently" since a couple months is still pretty recent and the exact date is unimportant (and we don't want to seem too tardy with the announcement).

goanpeca commented:
@CAM-Gerlach any chance of posting a screenshot? You can use a broswer extension to take a picture of the whole scrollable page (or use firefox which includes that already)

CAM-Gerlach commented:
@goanpeca

> any chance of posting a screenshot?

A screenshot of what? As I've mentioned before, the site fails to build locally for me (and @ccordoba12 as well) on multiple different Windows and Linux environments even carefully duplicating the exact steps that works on Travis, and other sites (from a simple one to the much more complicated Lektor homepage itself) all build fine.

goanpeca commented:
Ok, no screenshot then

CAM-Gerlach commented:
@goanpeca Just realized I never properly said thanks for the detailed review. So thanks :)

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+
+First up, we're thrilled to announce a major part of what's making that plan possible (along with your support, of course!).
+This May, Spyder was awarded a $3000 development grant from [NumFOCUS](https://www.numfocus.org/), an organization promoting better science through open code, to help with finishing Spyder 4!
+NumFOCUS is a nonprofit dedicated to supporting [key scientific computing projects](https://www.numfocus.org/sponsored-projects); [promoting sustainability](https://www.numfocus.org/programs/sustainability) in the open source ecosystem; educating scientists, engineers, developers and data analysts through their flagship [PyData](https://www.numfocus.org/programs/pydata) program; and enabling and  ...
```
Perhaps it should be better to say here

... the next generation *of scientists and engineers*

?

CAM-Gerlach replied:
Yup, great suggestion. Implemented.

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+This May, Spyder was awarded a $3000 development grant from [NumFOCUS](https://www.numfocus.org/), an organization promoting better science through open code, to help with finishing Spyder 4!
+NumFOCUS is a nonprofit dedicated to supporting [key scientific computing projects](https://www.numfocus.org/sponsored-projects); [promoting sustainability](https://www.numfocus.org/programs/sustainability) in the open source ecosystem; educating scientists, engineers, developers and data analysts through their flagship [PyData](https://www.numfocus.org/programs/pydata) program; and enabling and  ...
+
+We applied for the funding back in March of this year, along with many other scientific python projects, and were one of the fortunate projects selected after the review process.
```
`python` -> `Python`

I think it's better to capitalize it.

CAM-Gerlach replied:
Definitely; its a typo that I didn't. Done.

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+NumFOCUS is a nonprofit dedicated to supporting [key scientific computing projects](https://www.numfocus.org/sponsored-projects); [promoting sustainability](https://www.numfocus.org/programs/sustainability) in the open source ecosystem; educating scientists, engineers, developers and data analysts through their flagship [PyData](https://www.numfocus.org/programs/pydata) program; and enabling and  ...
+
+We applied for the funding back in March of this year, along with many other scientific python projects, and were one of the fortunate projects selected after the review process.
+The award will run from now through this November, and will enable multiple betas with new features to be released throughout 2018, with the first feature-complete release candidate planned by the year's end (we'll have more detail on all that in Part 3).
```
... from *June* through this November ...

ccordoba12 replied:
... with the first feature-complete release candidate *of Spyder 4* planned ...

CAM-Gerlach replied:
Done and done, and modified tense to match.

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+title: State of the Spyder, Part 2: Looking up
+---
+pub_date: 2018-06-01 00:00
```
Note to self: Update date before merging.

CAM-Gerlach replied:
Same with all of them.

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+## Quansight Hires Spyder Maintainer; Helps Fund Development
+
+Equally exciting, Spyder's maintainer, [Carlos Cordoba](https://github.com/ccordoba12), was recently hired by [Quansight](https://www.quansight.com/), the brand-new company headed by scientific Python luminary [Travis Oliphant](https://twitter.com/teoliphant?lang=en)!
+The father of much of the modern PyData ecosystem including NumPy, SciPy, and Numba, Travis has been instrumental in the dramatic growth of open-source data analysis with Python through founding or helping lead Enthought, Continuum/Anaconda, NumFOCUS, and more.
```
Remove comma after Scipy here:

... including NumPy, SciPy **,** and Numba

CAM-Gerlach replied:
Ah yes, the Oxford comma...I was long a fan, but I've (mostly) phased it out since I adopted AP style.

jitseniesen replied:
Disappointing. I was looking forward to a massive debate.

CAM-Gerlach replied:
Me? Vigorously debating some seemingly small writing or design issue? Who would ever expect that :rofl:

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+The father of much of the modern PyData ecosystem including NumPy, SciPy, and Numba, Travis has been instrumental in the dramatic growth of open-source data analysis with Python through founding or helping lead Enthought, Continuum/Anaconda, NumFOCUS, and more.
+Quansight, Travis' newest venture, focuses on scaling up the PyData ecosystem to solve even larger and more integrated problems, connecting companies with the open-source tools that can benefit their business, and helping build a more robust ecosystem for open-source scientific development.
+
+As part of that mission, Carlos was hired through at least the end of 2018 to help make several open-source projects more sustainable in the long run, and I'm sure you all will be happy to hear that one of them will be our very own Spyder!
```
... was hired *part-time* ...

ccordoba12 replied:
Sorry, I just saw that's mentioned below.

CAM-Gerlach replied:
:+1:

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+
+## Spyder 3.3.0 Ready to Go
+
+Meanwhile, on the development side of things, we're finishing the rollout of the next update in the stable Spyder 3 line, 3.3.0, over the next few days, and will have a full post shortly going over the changes and enhancements in detail.
```
I think `we're finishing` needs to be changed now for `we finished`

CAM-Gerlach replied:
Yeah, I guess so—changed to "we've just finished"

CAM-Gerlach replied:
Also, the tense of the whole para needed to be changed as well to match.

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+## Spyder 3.3.0 Ready to Go
+
+Meanwhile, on the development side of things, we're finishing the rollout of the next update in the stable Spyder 3 line, 3.3.0, over the next few days, and will have a full post shortly going over the changes and enhancements in detail.
+In brief, it'll have some significant under the hood revisions, cleaning house of some old end of life dependencies and legacy support (Python 3.3 and PyQt4/<5.5), and it'll separate the IPython Console code into a new ``spyder-kernels`` package to make it easier for you to work with different Python environments and remote kernels, and easier for us to add exciting new features down the road.
```
... the IPython Console *kernel* code ...

CAM-Gerlach replied:
Done.

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+
+Meanwhile, on the development side of things, we're finishing the rollout of the next update in the stable Spyder 3 line, 3.3.0, over the next few days, and will have a full post shortly going over the changes and enhancements in detail.
+In brief, it'll have some significant under the hood revisions, cleaning house of some old end of life dependencies and legacy support (Python 3.3 and PyQt4/<5.5), and it'll separate the IPython Console code into a new ``spyder-kernels`` package to make it easier for you to work with different Python environments and remote kernels, and easier for us to add exciting new features down the road.
+Additionally, it'll add new preferences options for console plotting and autocompletion, improve the usability and performance of Variable Explorer editors, add a new, fully automated error reporting feature to make submitting reports faster, easier and more helpful, and contain a number of other minor improvements and usability enhancements, along with numerous bug fixes.
```
... Variable Explorer *viewers* ...

(instead of `editors` which sounds a bit strange when talking about the Variable Explorer)

CAM-Gerlach replied:
I mean, technically they are editors, the changes relates specifically to the editing functionality, and they are called that in the source. However, per request changed to "object viewers."

ccordoba12 commented:
Great job @CAM-Gerlach, @goanpeca and @jitseniesen.

I left some minor comments, then this should be ready.

CAM-Gerlach commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+## Spyder Documentation, Reborn
+
+Aside from Spyder 3 and 4, we've got another big ticket item for you all in the next few days: our brand new docs!
+We've been working on a major overhaul of our documentation, which will see its first official release with Spyder 3.3.0.
```
Done.

CAM-Gerlach commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+Finally, we want to make sure you all can stay better informed of Spyder-related happenings going forward, as well as get the chance to interact directly with the Spyder team.
+
+We're planning on much more frequent announcements, status reports, news, and tutorials right here on our blog, as well as more timely updates on our social media, including our [Facebook page](https://www.facebook.com/SpyderIDE/notifications/) and [Twitter account](https://twitter.com/Spyder_IDE).
+Make to subscribe/follow if you'd like to stay up to date or want to get in touch; we only post important happenings, so we won't inundate your feeds with trivialities.
```
Silly typo; fixed.

CAM-Gerlach commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,91 @@
+Our [old docs](https://pythonhosted.org/spyder/) were hosted on PyPI/PythonHosted and, with the freezing of that service, couldn't be updated.
+Therefore, we've now moved to maintain them in a [new dedicated repo](https://github.com/spyder-ide/spyder-docs), build, deploy and host them ourselves with [Sphinx](http://www.sphinx-doc.org/en/master/), [doctr](https://drdoctr.github.io/doctr/) and [Github Pages](https://pages.github.com/) on a subdomain of our new site making them more convenient, responsive and accessible.
+
+![Preview of new Spyder documentation website, displaying a modern theme/layout and better screenshots](spyder-docs-preview.png)
```
Not sure, so I updated it.

CAM-Gerlach commented:
@ccordoba12 Thanks for your feedback; implemented. Updated the Spyder-docs screenshot, and also added a new one of Spyder 3.3 (showing the new remote kernel functionality).

ccordoba12 commented:
Ok, I'd say this is ready. @CAM-Gerlach or anyone else, any objections to merge and publish?

This would give us some days between this and the 3.3.1 post.

CAM-Gerlach commented:
@ccordoba12 Looks good on my end. I updated the date to today, and the already prepared Tweet and Facebook post can be scheduled for tomorrow as soon as this is pushed to production.

ccordoba12 commented:
Ok, merging and moving to production right away. Thanks to everyone involved!

CAM-Gerlach commented:
Thanks @ccordoba12 @goanpeca and @jitseniesen for all your help and feedback!

ccordoba12 commented on content/blog/spyder-status-2018-present/contents.lr:
```diff
@@ -0,0 +1,93 @@
+title: State of the Spyder, Part 2: Looking up
```
One last thing: is this right? Shouldn't this be

> The state of Spyder, Part 2

?

I mean, `the Spyder` sounds odd to me.

ccordoba12 replied:
I didn't notice that Part 1 was also named like this...

CAM-Gerlach replied:
Didn't we discuss this on Part 3?

ccordoba12 replied:
I don't remember. Let's merge it to be consistent with part one ;-)

CAM-Gerlach replied:
Yeah, that was what we decided there :)

CAM-Gerlach commented:
@ccordoba12 I'll double-check how it looks on ``develop`` before we merge to production.

CAM-Gerlach commented:
@ccordoba12 LGTM!

ccordoba12 commented:
Yep, it looks really nice. I pushed to production already.
